crates/cast/bin/cmd/wallet/list.rs

```diff
@@ -47,6 +54,7 @@ impl ListArgs {
             .mnemonic_indexes(Some(vec![0]))
             .trezor(self.trezor || self.all)
             .aws(self.aws || self.all)
+            .gcp(self.gcp || self.all)
```

Comment:
this makes me realize that if --all is set, the command will always fail if the env vars are missing, same for aws, which is probably not great UX.
we could make gcp infallible if the env vars are missing, same for aws.

Reply:
@mattsse  That's true. I created a helper function  `gcp_env_vars_set` to check that all GCP environment variables are set and fixed the above part to `.gcp(self.gcp || (self.all && gcp_env_vars_set()))`. 

https://github.com/foundry-rs/foundry/pull/8232/commits/e2f5814304310dd0611692a73729658f49745a13